client/cypress/end-to-end/ballot-comparison-audit-board.spec.js

```diff
@@ -0,0 +1,121 @@
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').click()
+    cy.findAllByText(/Upload successfully completed/).should('have.length', 2)
+    cy.wait(2000)
+    cy.findByText('Next').click()
+    cy.findAllByText('Target Contests').should('have.length', 2)
+    cy.get('input[type="checkbox"]').first().check({ force: true })
+    cy.findByText('Save & Next').click()
+    cy.findAllByText('Opportunistic Contests').should('have.length', 2)
+    cy.findByText('Save & Next').click()
+    cy.get('#state').select('AL')
+    cy.get('input[name=electionName]').type(`Test Election`)
+    cy.get('#risk-limit').select('10')
+    cy.get('input[name=randomSeed]').type("543210")
+    cy.findByText('Save & Next').click()
+    cy.wait(1000)
+    cy.logout()
+    cy.wait(2000)
+    cy.contains('Participating in an audit in your local jurisdiction?')
+    cy.loginJurisdictionAdmin('wtarkin@empire.gov')
+    cy.findByText(`Jurisdictions - TestAudit${id}`).siblings('button').click()
+    cy.wait(1000)
+    cy.fixture('CSVs/manifest/ballot_comparison_manifest.csv').then(fileContent => {
+      cy.get('input[type="file"]').first().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'ballot_comparison_manifest.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').spread((firstButton, secondButton) => {
+      firstButton.click()
+    })
+    cy.contains("Upload successfully completed")   
+
+    cy.fixture('CSVs/cvr/ballot_comparison_cvr.csv').then(fileContent => {
+      cy.get('input[type="file"]').last().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'ballot_comparison_cvr.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').click()
+    cy.findAllByText(/Upload successfully completed/).should('have.length', 2)
+    cy.wait(1000)
+    cy.logout()
+    cy.wait(2000)
+    cy.loginAuditAdmin('audit-admin-cypress@example.com')
+    cy.findByText(`TestAudit${id}`).click()
+    cy.findByText('Review & Launch').click()
+    cy.findByText('Launch Audit').click()
+    cy.findAllByText('Launch Audit').spread((firstButton, secondButton) => {
+      secondButton.click()
+    })
+    cy.wait(1000)
+    cy.logout()
+    cy.wait(2000)
+    cy.contains('Participating in an audit in your local jurisdiction?')
+
+  })
+  const board_credentials_url = ''
+
+  it('Audit Board - Setup', () => {
+    cy.loginJurisdictionAdmin('wtarkin@empire.gov')
+    cy.findByText(`Jurisdictions - TestAudit${id}`).siblings('button').click()
+    cy.wait(1000)
+    cy.contains('Number of Audit Boards')
+    cy.findByText('Save & Next').click()
+    cy.findByText('Download Audit Board Credentials').click()
+    cy.logout()
+    cy.wait(2000)
+  })
+
+  it('Audit Board - Login with Credentials', () => {
+    cy.task('getPdfContent', `cypress/fixtures/PDFs/Audit Board Credentials\ -\ Death Star\ -\ TestAudit${id}.pdf`).then((content) => {
```

Comment:
Yes, PDF parsing wasn't possible with Cypress, and so I used a node command to read & process, and as far as I could see, the only way to do that was via a task instead of a Cypress custom command.